core/src/test/java/org/apache/druid/java/util/metrics/StubServiceEmitter.java

```diff
@@ -37,7 +41,18 @@ public StubServiceEmitter(String service, String host)
   @Override
   public void emit(Event event)
   {
-    events.add(event);
+    if (event instanceof AlertEvent) {
+      final AlertEvent alertEvent = (AlertEvent) event;
+      log.warn(
+          "[%s] [%s] [%s]: %s%n",
+          alertEvent.getSeverity(),
+          alertEvent.getService(),
+          alertEvent.getFeed(),
+          alertEvent.getDescription()
```

Comment:
Pattern expects four string and a number; only four strings provided.

Reply:
The last one `%n` prints a newline.